WordPress/Classes/ViewRelated/System/WPTabBarController.m

```diff
@@ -274,9 +274,18 @@ - (ReaderCoordinator *)readerCoordinator
 
 - (NSArray<UIViewController *> *)tabViewControllers
 {
-    return @[self.mySitesCoordinator.rootViewController,
-             self.readerNavigationController,
-             self.notificationsSplitViewController];
+    NSArray *tabs = @[
+        self.mySitesCoordinator.rootViewController,
+        self.notificationsSplitViewController
+    ];
+    
+    NSMutableArray *mutableTabs = [tabs mutableCopy];
+    
+    if ([AppConfiguration showsReader]) {
```

Comment:
Nit: Since this is a property we should use dot notation. 

```suggestion
    if AppConfiguration.showsReader) {
```

Reply:
Fixed! 4535842